### PR TITLE
Feature/ai reviewer new config

### PR DESCRIPTION
**Pull request by yousiefc**
# Description

This PR includes the following proposed change(s):

- text area to submit new JSON config

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

```bash
yarn lint
yarn test
```

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/ai-reviewer-admin/src/App.js [hunk 1]
```diff
@@ -14,9 +15,9 @@ function App() {
   return (
     <>
       <Header header={header} />
-      <div className="content">
```
Comment (KDNeufeld) [inline src/frontend/ai-reviewer-admin/src/App.js:17]: This actually breaks the sticky footer.
App.scss has
.content {
  flex: 1 0 auto;
}
which makes the footer "stick" to the bottom when the page is shorter than the viewport.  The middle body auto expands to fill 100% vertical.

Comment (yousiefc) [reply src/frontend/ai-reviewer-admin/src/App.js:17]: Ah kk will revert. Thanks:)

Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/src/App.js:17]: or incorporate the content css class into the Container wrapper

File: src/frontend/ai-reviewer-admin/package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@bcgov/bootstrap-theme": "github:bcgov/bootstrap-theme",
+    "@material-ui/core": "^4.11.3",
```
Comment (KDNeufeld) [inline src/frontend/ai-reviewer-admin/package.json:7]: Good. Looking forward to this addition

File: src/frontend/ai-reviewer-admin/src/domain/documents/DocumentList.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ export default function DocumentList({ configurations }) {
         <span className="d-none d-sm-inline col-sm-3">Action</span>
       </div>
       <ul>
-        {configurations.map((configuration) => (
+        {configurations && configurations.map && configurations.map((configuration) => (
```
Comment (KDNeufeld) [inline src/frontend/ai-reviewer-admin/src/domain/documents/DocumentList.js:18]: I actually changed this in my PR.  I moved this logic to the service layer to throw an error if the result is not an actual map, in which case, this configurations object will either be null or a valid object.  Hopefully. :)

File: src/frontend/ai-reviewer-admin/src/App.js (hunk 1 above)
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/src/App.js:17]: Great. this works again.

**Review by KDNeufeld — APPROVED** [review]
Great work.

**Pull request merged**